docs/configuration/screen-share.md

```diff
@@ -0,0 +1,65 @@
+
+How to Share A Screen Across Qubes
+==================================
+
+## Setup The Shared Screen Server
+
+In the Qube you want to want to share a screen from (referred to the Content Qube) execute these commands:
+
+1. Install packages `sudo apt install -y xfwm4 tigervnc-standalone-server tigervnc-viewer`
+
+2. Start the server `vncserver :10  -xstartup /usr/bin/xfwm4 -geometry 1920x1080 -localhost no`
+
+3. Enter passwords when prompted
+
+    1. You can generate secure enough passwords for this purpose using `openssl rand -base64 16 | tr -d '+/=' | head -c 8;echo`
+    
+    2. Enter one password for read/write (first password and verify prompt)
+
+    3. Enter a different password for the view only password (second password and verify prompt)
+
+3. View the shared screen `xtigervncviewer -passwd /home/user/.vnc/passwd :10`
+
+4. Open applications `DISPLAY=:10 xterm` where xterm can be any binary on your system
+
+## Qubes Connect TCP Service
+
+These steps are a simpler version of [The Qubes Firewall](https://www.qubes-os.org/doc/firewall/#opening-a-single-tcp-port-to-other-network-isolated-qube). In dom0 execute these steps:
+
+1. Edit /etc/qubes-rpc/policy/qubes.ConnectTCP 
+
+2. Add the line: `PRESENTATION-QUBE @default ask,target=CONTENT-QUBE`
```

Comment:
Maybe replace:
 -  `PRESENTATION-QUBE` with `<PRESENTATION_QUBE>` 
 -  `CONTENT-QUBE` with <CONTENT_QUBE>`

to be consistent with the official docs. Also in other places of this document.